plane/source/docs/quadplane-transitions.rst

```diff
@@ -51,7 +51,8 @@ for transitions to QuadPlane modes while flying at high speed. Tilt-rotors will,
 - VTOL attitude control will be provided as needed as the vehicle slows.
 - Transition to altitude holding VTOL modes will manage power to the VTOL motors as necessary to hold altitude as the vehicle slows from forward fixed wing flight.
 - Transition to non-altitude holding VTOL modes will provide vertical thrust as commanded by the throttle.
-- Transition to QLOITER will project a stopping position to maintain based on deceleration of the vehicle and then hold it once reached.
+- Transition to position holding modes will project a stopping position to maintain based on deceleration of the vehicle and then hold it once reached.
+- When transitioning to a position holding mode, like QLIOTER, QuadPlane will try to decelerate, which can result in the nose pitching up rapidly if traveling at high speeds. In order to prevent this, the pitch is initially limited to 0 degrees, relaxing this limit to the smaller of :ref:`Q_ANGLE_MAX<Q_ANGLE_MAX>` or :ref:`LIM_PITCH_MAX<LIM_PITCH_MAX>` over the period of :ref:`Q_BACKTRANS_MS<Q_BACKTRANS_MS>`.
```

Comment:
It is active in all modes (except Qacro). Maybe add something like "Note that this limit is applied in all modes"

Reply:
done